minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java

```diff
@@ -327,6 +331,12 @@ public MiniAccumuloClusterImpl(MiniAccumuloConfigImpl config) throws IOException
 
     this.config = config.initialize();
 
+    if (Boolean.valueOf(this.config.getSiteConfig().get(Property.TSERV_NATIVEMAP_ENABLED.name()))) {
```

Comment:
Oh, I just realized. This is also buggy because you're using `.name()` instead of `.getKey()`. So, this will always return false. That's probably why you've never seen the native map failing to load that I mentioned in my other comment.

Reply:
Except that the code does work. I tested NativeMapIT in the IDE after running `mvn clean` on the filesystem and it failed.

Reply:
NativeMapIT is a special test. It sets a system property to load the native maps. It works differently than other mini tests.